schemas/examples/history_with_decoding_df5.json

```diff
@@ -0,0 +1,44 @@
+{
+  "data": {
+    "coordinates": "",
+    "timestamp": "1687922293",
+    "gw_mac": "C8:25:2D:8E:9C:2C",
+    "tags": {
+      "F4:1F:0C:28:CB:D6": {
+        "rssi": -51,
+        "timestamp": "1687922293",
+        "data": "0201061BFF990405167546CAC67C000CFFB403E07796A44D5FF41F0C28CBD6",
+        "dataFormat": 5,
+        "temperature": 28.745,
+        "humidity": 45.3050,
+        "pressure": 150812,
```

Comment:
The pressure seems higher than what DF5 can actually support, I think valid range is 50000 ... 50000 + 2^16-1.

I'd pick a value near 100 000 since that's usually a reasonable value near sea surface level